scripts/fetchSbomData.ts

```diff
@@ -0,0 +1,62 @@
+import fs from "fs";
+import fetch from "node-fetch";
+
+interface SBOMData {
+  dependencies?: Record<string, unknown>;
+  [key: string]: unknown; // Add more specific fields if known
+}
```

Comment:
are you sure the type is right? I don't see any `dependencies` keyword in the API's response.

You can use tools like JSON to TypeScript to easily generate types from JSON.

https://transform.tools/json-to-typescript

```suggestion
interface GitHubSbomApiResponse {
  sbom: {
    spdxVersion: string
    dataLicense: string
    SPDXID: string
    name: string
    documentNamespace: string
    creationInfo: {
      creators: string[]
      created: string
    }
    packages: {
      name: string
      SPDXID: string
      versionInfo: string
      downloadLocation: string
      filesAnalyzed: boolean
      licenseConcluded?: string
      copyrightText?: string
      externalRefs: {
        referenceCategory: string
        referenceType: string
        referenceLocator: string
      }[]
      licenseDeclared?: string
    }[]
    relationships: {
      spdxElementId: string
      relatedSpdxElement: string
      relationshipType: string
    }[]
  }
}
```